ansible_collections/arista/avd/roles/eos_designs/python_modules/ip_addressing/avdipaddressing.py

```diff
@@ -18,13 +18,13 @@ class AvdIpAddressing(AvdFacts, UtilsMixin):
     be overridden by a custom python class.
     """
 
-    def _ip(self, pool: str, prefixlen: int, subnet_offset: int, ip_offset: int) -> str:
+    def _ip(self, pool: str, prefixlen: int, subnet_offset: int, ip_offset: int, add_subnet_space: bool = True) -> str:
         pool_network = ipaddress.ip_network(pool, strict=False)
         prefixlen_diff = prefixlen - pool_network.prefixlen
         subnet_size = (int(pool_network.hostmask) + 1) >> prefixlen_diff
         if (subnet_offset + 1) * subnet_size > pool_network.num_addresses:
             raise AristaAvdError(f"Unable to get {subnet_offset + 1} /{prefixlen} subnets from pool {pool}")
-        subnet = ipaddress.ip_network((int(pool_network.network_address) + subnet_offset * subnet_size, prefixlen))
+        subnet = ipaddress.ip_network((int(pool_network.network_address) + subnet_offset * (subnet_size if add_subnet_space else 1), prefixlen))
```

Comment:
I don't understand why you need to change this. We should not need to modify this neutral function for this corner case